.changeset/rude-ravens-move.md

```diff
@@ -0,0 +1,6 @@
+---
+'@web/storybook-builder': major
+'@web/storybook-framerwork-web-components': major
```

Comment:
both have to be minor
we need to stay on 0.x, while this will do 1.x

0.x minor bump is sufficient to behave same as 1.x major bump